Sources/Target/Windows/Process.cpp

```diff
@@ -104,10 +122,52 @@ ErrorCode Process::detach() {
   return kSuccess;
 }
 
+BOOL Process::writeDebugBreakCode(uint64_t address) {
```

Comment:
It may or may not be useful to make `Process::writeDebugBreakCode` return an `ErrorCode` instead. That way you have slightly more info on what failed after calling `writeMemory` for instance.